mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -151,7 +151,8 @@ class CrossValidator extends Estimator[CrossValidatorModel] with CrossValidatorP
 @AlphaComponent
 class CrossValidatorModel private[ml] (
     override val parent: CrossValidator,
-    val bestModel: Model[_])
+    val bestModel: Model[_],
+    val crossValidationMetrics: Array[Double])
```

Comment:
Could you please add doc, such as ```Metric for each parameter setting tested during cross-validation, corresponding to the settings in `parent.estimatorParamMaps`. Each value is the average over numFolds trials.```

Also, do you think "crossValidationMetrics" is too long a name, especially since it's clear it has to do with cross-validation?  How about "avgMetrics?"